modules/project/variables.tf

```diff
@@ -302,10 +302,12 @@ variable "shared_vpc_service_config" {
   description = "Configures this project as a Shared VPC service project (mutually exclusive with shared_vpc_host_config)."
   # the list of valid service identities is in service-agents.yaml
   type = object({
-    host_project                = string
-    service_identity_iam        = optional(map(list(string)), {})
-    service_identity_subnet_iam = optional(map(list(string)), {})
-    service_iam_grants          = optional(list(string), [])
+    host_project                 = string
+    host_project_iam             = optional(list(string), [])
+    service_identity_iam         = optional(map(list(string)), {})
+    service_identity_subnets_iam = optional(map(list(string)), {})
+    service_iam_grants           = optional(list(string), [])
+    subnets_iam                  = optional(map(list(string)), {})
```

Comment:
same as above, only network user role makes sense from service projects, and for service identities we already support more roles